EarTrumpet/prebuild.ps1

```diff
@@ -2,6 +2,6 @@
 
 $PackageManifestPath = "$PSScriptRoot\..\EarTrumpet.Package\Package.appxmanifest"
 
-$xml = [xml](Get-Content $PackageManifestPath)
+$xml = [xml](Get-Content "$PackageManifestPath ")
```

Comment:
Can you trim that extra space on this line?

Reply:
Already ahead of you. I trimmed that extra space in the second commit https://github.com/File-New-Project/EarTrumpet/pull/525/commits/9704823165b6833cea52578ac795b700495ab87b